kurtosis-devnet/pkg/kurtosis/sources/spec/spec.go

```diff
@@ -7,16 +7,31 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+const (
+	FeatureInterop = "interop"
+)
+
 // ChainSpec represents the network parameters for a chain
 type ChainSpec struct {
 	Name      string
 	NetworkID string
 }
 
+type FeatureList []string
+
+func (f FeatureList) Contains(feature string) bool {
+	for _, f := range f {
```

Comment:
In the loop, the variable f shadows the method receiver f, which can be a little confusing. Maybe renaming the loop variable to something like feat could help with clarity?